test/experimental/features/eventtype_autocreation/eventtype.go

```diff
@@ -0,0 +1,77 @@
+/*
+Copyright 2021 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package eventtype_autocreation
+
+import (
+	"context"
+
+	"knative.dev/eventing/test/rekt/resources/eventtype"
+
+	"github.com/cloudevents/sdk-go/v2/test"
+	"k8s.io/apimachinery/pkg/util/sets"
+	sourcesv1 "knative.dev/eventing/pkg/apis/sources/v1"
+	"knative.dev/eventing/test/rekt/resources/broker"
+	"knative.dev/eventing/test/rekt/resources/pingsource"
+	"knative.dev/eventing/test/rekt/resources/trigger"
+	duckv1 "knative.dev/pkg/apis/duck/v1"
+	"knative.dev/reconciler-test/pkg/eventshub"
+	"knative.dev/reconciler-test/pkg/eventshub/assert"
+	"knative.dev/reconciler-test/pkg/feature"
+	"knative.dev/reconciler-test/pkg/manifest"
+	"knative.dev/reconciler-test/pkg/resources/service"
+)
+
+// SendsEventsWithEventTypes tests pingsource to a ready broker.
```

Comment:
```suggestion
// SendsEventsFromPingSourceWithEventTypes tests that the created EventType from a Pingsource to a Broker is of type "dev.knative.sources.ping".
```
Or something like this. But making it a bit clearer